### ISSUE TITLE
License page status bar on dark mode bugged

### ISSUE THREAD

**Issue by jolamar** [flutter/flutter#102155]
Using the LicensePage package in dark mode the status bar icons are black on black so they are not visible. It should honor the theme brightness.

**Comment by jolamar** [comment]
https://github.com/flutter/flutter/pull/102153

**Comment by maheshmnj** [comment]
@jolamar is this issue resolved?

**Comment by jolamar** [comment]
Yes we had to change a few values in our theme file that were causing the issue.
